vcd/config_test.go

```diff
@@ -235,20 +240,30 @@ func getConfigStruct() TestConfig {
 	// Now we fill the environment variables that the library is using for its own initialization.
 	if configStruct.Provider.TerraformAcceptanceTests {
 		// defined in vendor/github.com/hashicorp/terraform/helper/resource/testing.go
-		os.Setenv("TF_ACC", "1")
+		_ = os.Setenv("TF_ACC", "1")
+	}
+	if configStruct.Provider.MaxRetryTimeout == 0 {
+		// Setting a default value that should be reasonable for these tests, as we run many heavy operations
+		_ = os.Setenv("VCD_MAX_RETRY_TIMEOUT", "300")
```

Comment:
Could you double check that we are consistent in that test config values from JSON take priority over env variables?

Reply:
Already checked. We set the variables just before invoking the provider constructor.